docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -94,36 +94,58 @@ Running a single query
 To run a single query over a CodeQL database for a JavaScript codebase,
 you could use the following command from the directory containing your database::
 
-   codeql database analyze <javascript-database> ../ql/javascript/ql/src/Declarations/UnusedVariable.ql --format=csv --output=js-analysis/js-results.csv 
+   codeql database analyze <javascript-database> ../ql/javascript/ql/src/Declarations/UnusedVariable.ql --format=csv --output=js-analysis/js-results.csv
 
 This command runs a simple query that finds potential bugs related to unused
 variables, imports, functions, or classes---it is one of the JavaScript
 queries included in the CodeQL repository. You could run more than one query by
 specifying a space-separated list of similar paths.
 
 The analysis generates a CSV file (``js-results.csv``) in a new directory
-(``js-analysis``).  
+(``js-analysis``).
 
 You can also run your own custom queries with the ``database analyze`` command.
 For more information about preparing your queries to use with the CodeQL CLI,
 see ":doc:`Using custom queries with the CodeQL CLI <using-custom-queries-with-the-codeql-cli>`."
 
+Running a CodeQL pack
+~~~~~~~~~~~~~~~~~~~~~
+
+.. pull-quote::
+
+    Note
+
+    The CodeQL package manager is currently in beta and subject to change. During the beta, CodeQL packs are available only in the GitHub Package Registry (GHPR). You must use version 2.5.8 or later of the CodeQL CLI to use the CodeQL package manager.
+
+    During the beta, you must specify ``--allow-packs`` whenever you run the ``analyze`` command.
+
+To run an existing CodeQl pack from the GitHub Package Registry (GHPR), you need to download it first:
+
+   codeql pack download microsoft/coding-standards@1.0.0
+
+Afterwards, you can run the pack on a specific database:
+
+   codeql database analyze <database> --allow-packs microsoft/coding-standards@1.0.0 <scope>/<other-pack>
+
+The ``analyze`` command above runs the default suite from ``microsoft/coding-standards v1.0.0`` and the latest version of ``scope/other-pack`` on the specified database.
+For further information about default suites, see ":ref:`Publishing and using CodeQL packs <publishing-and-using-codeql-packs>`".
+
 Running GitHub code scanning suites
```

Comment:
I think these should be called _query suites_.

And again, separating out running queries, packs, and query suites into 3 sections seems artificial, since you can combine all three of them in a single cli invocation. I'd rather see a single section that includes a description of the different ways of running `database analyze`.

And then you can have a separate section (if necessary) that describes `github upload-results`

Reply:
To be clear, my suggestion above might be out of scope for this PR. If you don't have time to address this before the packaging release, then it's fine to think about it for later.

Reply:
This is a great suggestion. I was wondering how to best structure or group the different ways to analyze databases. I'm going to focus on fixing this PR from a functional perspective first and note this as a work item for the next iteration. ✨

Reply:
Once the diff is outdated, this comment thread might be lost, so I moved it over to the main thread:
https://github.com/github/codeql/pull/6339#issuecomment-886510387